conda/recipes/dask-cudf/meta.yaml

```
@@ -41,18 +41,27 @@ requirements:
     - dask >=2023.5.1
     - dask-core >=2023.5.1
     - distributed >=2023.5.1
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
   run:
     - python
     - cudf ={{ version }}
     - dask >=2023.5.1
     - dask-core >=2023.5.1
     - distributed >=2023.5.1
+    {% if cuda_major == "11" %}
     - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+    {% endif %}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
 
 test:
   requires:
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
```

Is `cuda-version` sufficient for constraining `cudatoolkit` ([ref]( https://github.com/conda-forge/cuda-version-feedstock/blob/3ce1bae701079226f982b173667edc5d0a64c01f/recipe/meta.yaml#L17 ))?

```suggestion
    - cudatoolkit
```